### ISSUE TITLE
Card "Your order" on checkout page displayed in wrong language

### ISSUE THREAD

**Issue by korry73** [woocommerce/woocommerce#17278]
## Prerequisites

- [X ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [ ] The issue still exists against the latest `master` branch of WooCommerce on Github (this is **not** the same version as on WordPress.org!)
- [X] I have attempted to find the simplest possible steps to reproduce the issue
- [X] I have included a failing test as a pull request (Optional)

## Steps to reproduce the issue

1. Install Wordpress 3.2.1, Woocommerce, Polylang
2. Enable Bulgarian and English languages with Bulgarian being the default
3. Login to the frontend and switch the language to English
4. Add a product to the card
5. Proceed to checkout page
6. The Your Order card is initially displayed correctly in English, but shortly after that it reloads and several fields are displayed in Bulgarian

## Expected/actual behavior

Your Order section should be displayed in English (or another non-main language)

## Isolating the problem

- [ ] This bug happens with only WooCommerce plugin active
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/)
- [X] I can reproduce this bug consistently using the steps above

## WordPress Environment

<details>

WordPress environment
--

WordPress environment
--
Home URL: |   | https://zornica.info/wptest2
Site URL: |   | https://zornica.info/wptest2
WC version: |   | 3.2.1
Log directory writable: |   | /home/zornica/public_html/wptest2/wp-content/uploads/wc-logs/
WP version: |   | 4.8.2
WP multisite: |   | –
WP memory limit: |   | 256 MB
WP debug mode: |   | –
WP cron: |   |  
Language: |   | en_GB

![incorrect](https://user-images.githubusercontent.com/32910820/31738842-8dcb97fe-b454-11e7-8e7c-23567d428be4.png)

```
</details>

**Comment by mikejolley** [comment]
See https://wordpress.org/support/topic/updating-translations-after-updates/

tl;dr all the strings are localised, it's just missing in your mo/po file. Remember strings are case sensitive.

**Comment by korry73** [comment]
Hi Mike, thanks for your reply. But I am not experiencing the issue you describe, i have the opposite problem, labels that should be displayed in English get displayed in Bulgarian (and I have them in my PO / MO files). The problem appeared after an update to Woocommerce from 3.1.2 to 3.2. I attach a screenshot from WC 3.1.2 with a correct behaviour.
![correct](https://user-images.githubusercontent.com/32910820/31756693-f3429bc4-b4ad-11e7-8090-622108cdcf05.png)

As you see here, the labels for **Total, Subtotal** etc as well as the **Place order** label on the button below are displayed correctly.

**Comment by ktheodor** [comment]
I have the same problem
Woocommerce version 3.2.1 checkout page, after load does an extra call to ?wc-ajax=update_order_review which appears to switch the order review and payments section back to site default language.

**Comment by korry73** [comment]
As a workaround, you can build your URLs with a language prefix: domain.tld/LANG_PREFIX/checkout. Then it works correctly. However if you skip the lang prefix for the default language, you get into the issue for the rest of the languages.

**Comment by mikejolley** [comment]
Using WPML or similar? You should definitely check the config of those types of plugin. WC does not switch languages.
